timelock-impl/src/test/java/com/palantir/atlasdb/timelock/paxos/LocalPaxosComponentsTest.java

```diff
@@ -148,4 +147,12 @@ public LocalPaxosComponents createPaxosComponents(
                 timeLockVersion,
                 false);
     }
+
+    private static File getAndCreateSubdirectory(File base, String subdirectoryName) {
+        File file = base.toPath().resolve(subdirectoryName).toFile();
+        if (file.mkdirs()) {
+            return file;
+        }
+        throw new RuntimeException("Unexpected error when creating a subdirectory");
+    }
```

Comment:
We duplicate this function also in another package.

Reply:
Does it make sense to put this to a common package. Though I couldn't find any package for common utils.

Reply:
I wouldn't be opposed to having it under `atlasdb-tests-shared` on a utils package? But fine if we want to leave this one duplication here, since most other uses have been refactored to the use the `SubdirectoryCreator`. If, as we move more packages, we find this code being needed to be duplicated even more times, then wwe can more to the tests-shared package.